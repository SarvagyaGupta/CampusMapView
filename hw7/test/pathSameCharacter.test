# Find a path with from a character in the shortened Marvel dataset to the same character.

# To run this test, you will need to have downloaded the shortMarvel dataset and
# place it in src/hw7/data/shortMarvel.tsv.

LoadGraph shortMarvel shortMarvel.tsv

FindPath shortMarvel FLASH FLASH
