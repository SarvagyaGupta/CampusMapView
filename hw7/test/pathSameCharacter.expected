# Find a path with from a character in the shortened Marvel dataset to the same character.

# To run this test, you will need to have downloaded the shortMarvel dataset and
# place it in src/hw7/data/shortMarvel.tsv.

loaded graph shortMarvel

path from FLASH to FLASH:
total cost: 0.000
